src/Components/CreateImageWizard/steps/index.js

```diff
@@ -0,0 +1,8 @@
+export { default as awsTarget } from './aws';
```

Comment:
What is the benefit of importing and exporting the steps in this file instead of just importing them in `CreateImageWizard.js`?

Reply:
You can import all steps as
```JS
import { review, awsTarget, registration, googleCloudTarget, msAzureTarget, packages, imageOutput } from './steps';
```

Instead of importing one by one
```JS
import review from './steps/review';
import awsTarget from './steps/awsTarget';
import registration from './steps/registration';
import googleCloudTarget from './steps/googleCloudTarget';
import msAzureTarget from './steps/msAzureTarget';
import packages from './steps/packages';
import imageOutput from './steps/imageOutput';
```